duckduckgo/Android
# ADS: Bookmarks migration

Task/Issue URL: https://app.asana.com/0/0/1203411647299662/f

### Description
Migrate Bookmarks section to new ADS components

### Steps to test this PR

- Install from this branch
- Go to Bookmarks

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20221213_102855_DuckDuckGo](https://user-images.githubusercontent.com/20798495/207294198-efb78249-b069-4a3d-a4ee-5f41ff8044f9.jpg)|![Screenshot_20221213_101924_DuckDuckGo](https://user-images.githubusercontent.com/20798495/207294348-58141c01-061a-4110-b134-12d7eba25224.jpg)|
![Screenshot_20221213_102953_DuckDuckGo](https://user-images.githubusercontent.com/20798495/207294220-4b51fb2c-51fa-4a98-8a89-01f5982ff1d8.jpg)|![Screenshot_20221213_101956_DuckDuckGo](https://user-images.githubusercontent.com/20798495/207294301-fcd41476-a7e5-4dc4-a6c1-90920bb35321.jpg)|
